### PR TITLE
Add release topic entry for Java dump time zone update

### PR DESCRIPTION
**Pull request by SueChaplain**
- Include an entry in the release topic for this update

Signed-off-by: SueChaplain <Sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Added a what's new entry for your Java dump time zone update @keithc-ca . If you are happy with it, please merge.

File: docs/version0.27.md [hunk 1]
```diff
@@ -27,15 +27,19 @@
...
 
-### Performance improvements
 
 ### New `-XX:[+|-]AdaptiveGCThreading` option added
 
 To optimize performance, you can now enable adaptive threading to automatically tune the number of active parallel garbage collection (GC) threads. When enabled, the GC thread count is dynamically adjusted from collection cycle to cycle to account for changes in the workload and consider the available resources. When parallel workloads decrease, less threads can be used, reducing the overhead and freeing up resources for other processing activities.
 
 Use the [`-xgcmaxthreads`](xgcmaxthreads.md) option with the [`-XX:+AdaptiveGCThreading`](xxadaptivegcthreading.md) option to specify a thread count limit.
 
+### Improved time zone information added to a Java dump file
```
Comment (keithc-ca) [inline docs/version0.27.md:41]: I would say '... added to Java dump files' (it's added to all such files, not just one).

Is this the target of the reference on line 30? If so, that reference needs to be updated accordingly.

Comment (SueChaplain) [reply docs/version0.27.md]: Updated as suggested (in all relevant places)

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
Jenkins doc stage

**Pull request merged**